RocketChat/Rocket.Chat.ReactNative
# fix: don't re-encode public key when changing e2e password

## Proposed changes


When the user changes their E2E password, their public key is sent in corrupted form to the server (sent as string instead of object)

## Issue(s)	

Closes: #4803 
Resolve: 5190

## How to test or reproduce


Use the mobile app to change E2E password by opening the side menu, selecting Settings -> Security and privacy -> E2E Encryption, entering a new password and clicking "Save Changes"

Then check using mongo that the public key of the user has been saved as a string (starts with a double quote (`"`) character) instead of an object (starting with a brace (`{`) character).

## Screenshots

## Types of changes




## Checklist



## Further comments
